app/models/service_provider.rb

```diff
@@ -70,6 +70,11 @@ def skip_encryption_allowed
     @allowed_list.include? issuer
   end
 
+  def biometric_ial_allowed?
+    IdentityConfig.store.biometric_ial_enabled &&
+      IdentityConfig.store.allowed_biometric_ial_providers.include?(issuer)
```

Comment:
in the past we have tried to manage per-SP things like this on the service providers table itself by adding a new column, etc. Is that something we considered this time?

Reply:
Yes, we considered it. The reasons we went with this approach are:
1. Speed of implementation
2. This will have a short life

Reply:
➕ to what vraj said, and also as a longer historical context: michael and i suggested this during the initial implementation, and were given a definitive "no" about it then.